### ISSUE TITLE
Convert Markdown to C# instead of Xaml

### ISSUE THREAD

**Issue by berggrenmille** [Trust-Anchor-Group/NeuroAccessMaui#101]
Directly creating Controls, using C# Markup or objects has performance benefits as parsing Xaml is slow. We also skip the step of generating and parsing the xaml, which is just extra work

**Comment by PeterWaher** [comment]
Good idea. The renderer can generate the objects dynamically instead of generating XAML in text form.
